data/speeds.yml

```diff
@@ -77,4 +77,13 @@ speeds:
         - 12
       13:
         - 5
-        - 11
\ No newline at end of file
+        - 11
+  slayer:
+    key: slayer
+    code: 8
+    shortName: SL
+    Description: Slayer
```

Comment:
Can you update these key names to be lowercase?

Reply:
I was looking at your validator, and it was just the upodate I was doing 🗡️